WordPress/Classes/ViewRelated/Blog/Blogging Reminders/BloggingRemindersFlowSettingsViewController.swift

```diff
@@ -252,8 +252,21 @@ class BloggingRemindersFlowSettingsViewController: UIViewController {
     @objc private func notifyMeButtonTapped() {
         tracker.buttonPressed(button: .continue, screen: .dayPicker)
 
-        let flowCompletionVC = BloggingRemindersFlowCompletionViewController(tracker: tracker)
-        navigationController?.pushViewController(flowCompletionVC, animated: true)
+        InteractiveNotificationsManager.shared.requestAuthorization { [weak self] allowed in
+            guard let self = self else {
+                return
+            }
+
+            DispatchQueue.main.async {
+                if allowed {
+                    let flowCompletionVC = BloggingRemindersFlowCompletionViewController(tracker: self.tracker)
+                    self.navigationController?.pushViewController(flowCompletionVC, animated: true)
```

Comment:
Nitpicky, but maybe we can take the `pushViewController` line out of the `if` block, since it's called in both paths - the conditional blocks seems to only be selecting which one the next VC is going to be.

Reply:
Yup, good improvement 👍